spec/schema.json

```diff
@@ -550,6 +550,24 @@
           "start_date",
           "is_on_sanctions_list"
         ],
+        "anyOf": [
+          {
+            "required": [
+              "beneficial_owner_nature_of_control_types"
+            ]
+          },
+          {
+            "required": [
+              "trustees_nature_of_control_types",
+              "trust_ids"
```

Comment:
Just wonder if 'trust_ids' should feature here (I took this from your previous suggestion @mouhajer-ch)?

Due to the 'anyOf' clause above, is it saying that the client could provide only some trust ids and no NOC type for the Individual BO, because that doesn't sound right?

Reply:
I 'think' by having the trustees and trust_ids in the same list it is joining them with an 'and'. So if they provide a trustee noc type than it must also have trust_ids values on the BO individual. Which sounds ok.

Reply:
It looks good to me, as I have suggested here: https://github.com/companieshouse/overseas-entities-api/pull/99#discussion_r978441233

Reply:
Thanks both. Will merge.